app/models/user.rb

```diff
@@ -123,6 +123,12 @@ def send_request_to_patron_creator_service
         }
       ],
     }
+    Rails.logger.debug(
+       {
+        'status' => 'Request send to patron creator service',
+        'dataSent' => query
```

Comment:
I'd rather not send the query to the log without passing it through a safety filter first, but we can fix this after we erplace the logging with our wrapper.

Reply:
I agree.